### DISCUSSION TITLE
[Script request]  Ryot

### DISCUSSION THREAD

**Discussion by Jmanko16** [community-scripts/ProxmoxVE#1011] [Request script]
### Application Name

Ryot

### Website

https://github.com/IgnisDa/ryot?tab=readme-ov-file

### Description

Roll your own tracker allows you to track media such as books, audiobooks, movies, fitness etc.  Has some overlap with an app such as trakt.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by 19Fidel84** [comment]
I would also be interested; the project looks good.

**Comment by kollock** [comment]
Bumping this request. I always have trouble with docker deployments that involve associated databases, so I've tried and failed for forever to get ryot online. Would love a helper script!
